modules/Finance/moduleFunctions.php

```diff
@@ -1196,15 +1194,15 @@ function receiptContents($guid, $connection2, $gibbonFinanceInvoiceID, $gibbonSc
             } else {
                 try {
                     $dataPayment = array('foreignTable' => 'gibbonFinanceInvoice', 'foreignTableID' => $gibbonFinanceInvoiceID);
-                    $sqlPayment = "SELECT gibbonPayment.*, surname, preferredName FROM gibbonPayment JOIN gibbonPerson ON (gibbonPayment.gibbonPersonID=gibbonPerson.gibbonPersonID) WHERE foreignTable=:foreignTable AND foreignTableID=:foreignTableID LIMIT $receiptNumber, 1";
+                    $sqlPayment = "SELECT gibbonPayment.*, surname, preferredName FROM gibbonPayment LEFT JOIN gibbonPerson ON (gibbonPayment.gibbonPersonID=gibbonPerson.gibbonPersonID) WHERE foreignTable=:foreignTable AND foreignTableID=:foreignTableID LIMIT $receiptNumber, 1";
                     $resultPayment = $connection2->prepare($sqlPayment);
                     $resultPayment->execute($dataPayment);
                 } catch (PDOException $e) {
                     $paymentFail = true;
                     $return .= "<div class='error'>".$e->getMessage().'</div>';
                 }
 
-                if ($resultPayment->rowCount() != 1) {
+                if ($resultPayment->rowCount() <= 0) {
```

Comment:
I've traditionally used !=1 because multiple records should not be returned, but if they are for some reason, an error seems better than grabbing the first record. Any thoughts?

Reply:
Okay, there's already a LIMIT 1 on the query, so in this case there shouldn't be a possibility of multiple records. I'll restore the original code.